Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -3768,6 +3780,24 @@ static inline bool isNotRunningInTestRunner()
     ASSERT_NOT_REACHED();
 }
 
+#if PLATFORM(MAC)
+void WebExtensionContext::runOpenPanel(WKWebView *, WKOpenPanelParameters *parameters, void (^completionHandler)(NSArray *URLs))
```

Comment:
```suggestion
void WebExtensionContext::runOpenPanel(WKWebView *, WKOpenPanelParameters *parameters, void (^completionHandler)(NSArray<NSURL *> *URLs))
```

Reply:
The genetics cause issues with C++, so I've been avoiding them in WebKit code.